README.md

```diff
@@ -232,10 +233,10 @@ Im Ordner `utilitis` finden sich Mixins, die in Komponenten eingesetzt werden k
 * _responsify.css.sass
 * …
 
-## Selector Best Practices
+## Best Practices
 
 * [Customizing Components](customizing-components.md)
-* [Selector Intent](http://cssguidelin.es/#selector-intent)
```

Comment:
Ich finde den Abschnitt der cssguidelin.es sehr gelungen! Unser plattes "In 99% der Fälle wird man generische Selektoren überschreiben (müssen), was zu mehr Code und einem erhöhtem Wartungsaufwand führt. Darum sei spezifischer und benutze stattdessen besser eine Klasse." ist zwar auch richtig, trifft aber nicht den Kern der Sache. Ich möchte mich exakt ausdrücken und die richtige Intention abbilden.

Vielleicht übersetzt du einfach das Beispiel in unsere Struktur?

Reply:
Ich verstehe dich. Letztendlich lässt es sich auf folgenden Satz herunterkochen (und das ist eigentlich, was ich sagen will): Sobald ein Element gestylet werden soll, setze eine Klasse.

Reply:
Ich bin mir nicht sicher, ob du mich verstehst. Ich finde Beispiele einfach sehr einleuchtend und sie sind zudem attraktiver für den flüchtigen Leser als Texte. Die aus den Verlinkungen fehlen mir jetzt.

Reply:
Die ersten beiden Selektorregeln, insbesondere die zweite, zeigen doch anhand von Beispielen genau das auf.